chapters/14.xml

```diff
@@ -3185,8 +3185,8 @@ Note that the
         <gloss>I give the book and [later]</gloss>
       </interlinear-gloss>
       <interlinear-gloss>
-        <jbo>… lebna lo rupnu vau do</jbo>
-        <gloss>… take some currency-units - from/to-you.</gloss>
+        <jbo>… lebna lo jdini vau do</jbo>
+        <gloss>… take some money - from/to-you.</gloss>
         <natlang>I give you the book and then take some dollars (pounds, yen) from you.</natlang>
```

Comment:
I suggest to change the natlang line too: ... take some money from you.

Reply:
fixed